dll/shellext/netshell/lang/ro-RO.rc

```diff
@@ -4,7 +4,8 @@
  * PURPOSE:     Romanian resource file
  * TRANSLATORS: Copyright 2011 Petru Dimitriu <petrimetri@gmail.com>
  *              Copyright 2011-2018 Ștefan Fulea <stefan.fulea@mail.com>
- *              Copyright 2023-2024 Andrei Miloiu <miloiuandrei@gmail.com>
+ *              Copyright 2023-2025 Andrei-Valentin Miloiu <miloiuandrei@gmail.com>
+ *              Copyright 2025 Petru Răzvan < >
```

Comment:
If no mail, then remove the ` < >`
(Note: this is unrelated to the build error)